### PR TITLE
feat: [CDS-35302]: pdc connector icon

### PR DESCRIPTION
**Pull request by azharness**
You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- StyleLint: `retrigger stylelint`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- ImageSnapshot `retrigger ImageSnapshot`

### CODE REVIEW

**Review by vardanbansal-harness — APPROVED** [review]
Make sure svg is optimized, using svgo.

File: src/icons/pdc.svg [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<svg width="27" height="29" viewBox="0 0 27 29" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (meenakshi-harness) [inline src/icons/pdc.svg:1]: Remove height, width and fill= 'none'

**Review by meenakshi-harness — CHANGES_REQUESTED** [review]

File: src/icons/pdc.svg (hunk 1 above)
Comment (knagurski) [inline src/icons/pdc.svg:1]: Beyond the removal of the height, width and fill, this SVG needs optimised. I use [ImageOptim](https://imageoptim.com)

Comment (azharness) [reply src/icons/pdc.svg]: Thanks! Fixed.

Comment (azharness) [reply src/icons/pdc.svg]: Thanks! Fixed.

**Review by meenakshi-harness — APPROVED** [review]

**Pull request merged**